src/OrbitGl/TrackHeader.cpp

```diff
@@ -42,53 +42,24 @@ void TrackHeader::DoDraw(PrimitiveAssembler& primitive_assembler, TextRenderer&
 
   const float x0 = GetPos()[0];
   const float y0 = GetPos()[1];
-  const float track_z = GlCanvas::kZValueTrack;
+  const float track_z = GlCanvas::kZValueTrackHeader;
   const float text_z = GlCanvas::kZValueTrackText;
 
   // Draw tab background
-  const float label_height = GetHeight();
-  const float half_label_height = 0.5f * label_height;
+  const float label_height = GetParent()->GetHeight();
   const float label_width = GetWidth();
-  const float half_label_width = 0.5f * label_width;
 
   const float indentation_x0 =
       x0 + (track_->GetIndentationLevel() * layout_->GetTrackIndentOffset());
-  Quad box = MakeBox(Vec2(indentation_x0, y0), Vec2(label_width, label_height));
-  primitive_assembler.AddBox(box, track_z, track_->GetTrackBackgroundColor(), shared_from_this());
 
-  // Early-out: In picking mode, don't draw the text and rounded corners.
-  if (picking) {
-    return;
+  if (track_->GetIndentationLevel() == 0 && layout_->GetDrawTrackHeaderBackground()) {
+    Quad box = MakeBox(Vec2(indentation_x0, y0), Vec2(label_width, label_height));
+    primitive_assembler.AddBox(box, track_z, track_->GetTrackBackgroundColor(), shared_from_this());
```

Comment:
What do you think about choosing a slightly different color for the TrackTab to differentiate it with its content?

Reply:
Yeah that would be cool, would have been easy to set colors with ImGui, but ImGui is not there anymore. Let's do this in an other PR.